release_docs/RELEASE.txt

```diff
@@ -314,12 +315,17 @@ Bug Fixes since HDF5-1.10.10 release
 
     Tools
     -----
-    - Names of objects with square brackets will have trouble without the
-      special argument, --no-compact-subset, on the h5dump command line.
+    - Added option --no-compact-subset to h5diff
 
-      h5diff did not have this option and now it has been added.
+      When the name of an object contains square brackets, h5diff was not able
+      to parse it correctly because “[ ]” were interpreted as the compact form
+      of subsetting.
 
-      Fix for GitHub issue #2682
+      h5dump has the special argument, --no-compact-subset to allow the use of
+      ‘[‘ and ‘]’ in dataset names.  h5diff now has this option as well.
+
```

Comment:
extra line?

Reply:
Removing line and fixing spelling of occurred.